README.md

```diff
@@ -44,18 +60,29 @@ lb_public_url = http://xxx.xxx.xxx.xxx
 
 > The application is being deployed to the compute instances asynchronously, and it may take a couple of minutes for the URL to serve the application.
 
-## Questions
+### MuShop Basic Topology
 
-If you have an issue or a question, please take a look at our [FAQs](./deploy/basic/FAQs.md) or [open an issue](https://github.com/oracle/oci-quickstart-cloudnative/issues/new).
+The following diagram shows the topology created by this stack.
+
+![MuShop Basic Infra](./images/basic/00-Free-Tier.png)
+
+## Getting Started with MuShop Complete
 
-## Deployment Topology
+MuShop Complete is a polyglot micro-services application built to showcase a cloud native approach to application development on [Oracle Cloud Infrastructure (OCI)][oci] using Oracle's [cloud native](https://www.oracle.com/cloud/cloud-native/) services. MuShop Complete uses a Kubernetes cluster, and can be deployed using the provided `helm` charts (preferred), or Kubernetes manifests. It is recommended to use an Oracle Container Engine for Kubernetes cluster, however other Kubernetes distributions will also work.
+
+The [helm chart documentation][chartdocs] walks through the deployment process and various options for customizing the deployment  
+
+###  MuShop Complete Topology
 
 The following diagram shows the topology created by this stack.
 
-![MuShop Basic Infra](./images/basic/00-Topology.png)
```

Comment:
MuShop Basic Infra pointing to complete topology folder

Reply:
its pointing to the right image and folder now.